src/shared/pipes.tsx

```diff
@@ -48,3 +48,16 @@ export const endDatePipe: Function = (startDate: Date, durWeeks: number): string
   endDate.setDate(endDate.getDate() + durWeeks * 7);
   return endDate.toLocaleDateString();
 };
+
+// pipes to add;
+// Empty string pipe
```

Comment:
change to short description of pipe

Reply:
Will go in an add these changes shortly, today or tomorrow